src/NUnitFramework/framework/Constraints/CollectionItemsEqualConstraint.cs

```diff
@@ -34,10 +34,11 @@ namespace NUnit.Framework.Constraints
     /// </summary>
     public abstract class CollectionItemsEqualConstraint : CollectionConstraint
     {
+
         /// <summary>
         /// The NUnitEqualityComparer in use for this constraint
         /// </summary>
-        private readonly NUnitEqualityComparer _comparer = new NUnitEqualityComparer();
+        protected NUnitEqualityComparer Comparer { get; } = new NUnitEqualityComparer();
```

Comment:
Mostly curious, why this change and the consequences below?

Reply:
Shadowing the two methods required making this accessible to child classes, otherwise the two shadowed methods would just have to throw an exception.